configure.ac

```diff
@@ -177,8 +177,8 @@ AX_OPTION_GATTRIB
 AX_DATA_DIRS
 # Where should PCB footprints be searched for?
 AX_PCB_DIRS
-# Set up libgeda with the correct ld version number
-AX_LIBGEDA([45:0:0])
+# Set up liblepton with the correct ld version number
+AX_LIBLEPTON([45:0:0])
```

Comment:
@vzh Yes, I think it is best to do it in this PR, so there's not a chance of anyone compiling & installing a liblepton.45.0.0 on their system.